index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Abdul Ali Noori's portfolio" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section>
+        <nav class="top-navigation">
+            <div class="welcome-message">Welcome</div>
+            <img src="images/Icons/Icon-Menu.svg" alt="GitHub">
+        </nav>
+        <header>
+            <div class="headline">
+                <p class="greeting-text">Hey there. I’m <br> Abdul Ali.</p>
+                <p class="profission-text">I’m a software developer</p>
+                <p class="supporting-text">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="social-media">
+                    <a href=""><img src="images/Icons/Icon-GitHub.svg" alt="GitHub"></a>
+                    <a href=""><img src="images/Icons/Icon-Linkedin.svg" alt="Linkedin"></a>
+                    <a href=""><img src="images/Icons/Icon-Angelist.svg" alt="Angelist"></a>
+                    <a href=""><img src="images/Icons/Icon-Twitter.svg" alt="Twitter"></a>
+                    <a href=""><img src="images/Icons/Icon-Medium.svg" alt="Medium"></a>  
+                </div>
+            </div>
+        </header>
+    </section>
```

Comment:
- [ ] Please remove these section tags surrounding the entire project. You need to have two sections.
- The header section, which I have highlighted above, and the headline section. :100:

Reply:
Hi @NahnahAJ,
thanks for your nice comment.
instead, what should I do?
Should I put my whole project into a tag? or just leave them separately in the body?